docs/LocalDevelopmentGuide.md

```diff
@@ -175,6 +175,9 @@ Request URL, submit the ngrok HTTPS URL with `/slack/commands` appended to the
 end. For the actual command, anything will work, though the final app will use
 `/rocket`.
 
+Select the checkbox titled "Escape channels, users, and links sent to your app"
+if you wish to escape usernames, channels, and links.
```

Comment:
Maybe make a note that you must check this box to have everything work properly

Reply:
Agreed - this is a requirement for the app, and some things will break if this option isn't enabled.